### PR TITLE
added NLP faculty 

### PR DESCRIPTION
**Pull request by bplank**
# Contributing to CSrankings
Added missing full-time NLP faculty in Europe (Bar-Ilan, Groningen, Rome)

**Inclusion criteria**

- [X] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [X] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**; please **do not** use the numeric suffixes, which are being obsoleted); include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

### CODE REVIEW

File: csrankings-f.csv [hunk 1]
```diff
@@ -55,6 +55,7 @@ Fabrizio Ferrandi,Politecnico di Milano,http://home.deib.polimi.it/ferrandi,_woi
 Fabrizio Granelli,University of Trento,https://www.granelli-lab.org/staff/fabrizio-granelli,Qk5Ni_IAAAAJ
 Fabrizio Luccio,University of Pisa,http://www.di.unipi.it/~luccio,NOSCHOLARPAGE
 Fabrizio Maria Maggi,University of Tartu,https://www.etis.ee/Portal/Persons/Display/a37d438b-3add-4c4f-8e49-dfacab570f23,NOSCHOLARPAGE
+Fabrizio Silvestri,Sapienza University of Rome,pi985dQAAAAJ,pi985dQAAAAJ
```
Comment (emeryberger) [inline csrankings-f.csv]: No web page (instead, duplicate Google Scholar ID).

File: csrankings-t.csv [hunk 2]
```diff
@@ -719,6 +719,7 @@ Tomer Koren,Tel Aviv University,https://tomerkoren.github.io,wGG1voYAAAAJ
 Tomer Michaeli,Technion,https://tomer.net.technion.ac.il,n2EbR2cAAAAJ
 Tomer Shlomi,Technion,http://www.cs.technion.ac.il/~tomersh,NOSCHOLARPAGE
 Tomi Männistö,University of Helsinki,https://www.cs.helsinki.fi/u/tomimann,ZSX8_ToAAAAJ
+Tommaso Caselli,University of Groningen,https://www.researchgate.net/profile/Tommaso-Caselli,fxQvP_QAAAAJ
```
Comment (emeryberger) [inline csrankings-t.csv]: Not clearly in CS (see inclusion criteria).

File: csrankings-g.csv [hunk 3]
```diff
@@ -583,6 +583,7 @@ Gorjan Alagic,University of Maryland - College Park,http://www.alagic.org,NOSCHO
 Gorkem Serbes,Yıldız Technical University,https://www.ce.yildiz.edu.tr/personal/gorkem?lang=en,96vDpfcAAAAJ
 Görschwin Fey,Hamburg University of Technology,https://www.tuhh.de/es/ce/people/gfey.html,NOSCHOLARPAGE
 Gorur Narayana Srinivasa Prasanna,IIIT Bangalore,http://www.iiitb.ac.in/faculty_page.php?name=GNSrinivasaPrasanna,NOSCHOLARPAGE
+Gosse Bouma,University of Groningen,http://www.let.rug.nl/gosse/,sd-w59cAAAAJ
```
Comment (emeryberger) [inline csrankings-g.csv]: Not clearly in CS (see inclusion criteria).

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings-f.csv (hunk 1 above)
Comment (bplank) [reply csrankings-f.csv]: fixed (sorry my bad)

File: csrankings-t.csv (hunk 2 above)
Comment (bplank) [reply csrankings-t.csv]: linked to RUG webpage that shows CS

File: csrankings-g.csv (hunk 3 above)
Comment (bplank) [reply csrankings-g.csv]: removed

**Comment by bplank** [comment]
updated requested files

**Review by bplank — COMMENTED** [review]
Thanks for spotting, I've updated the pull request.

**Review by emeryberger — COMMENTED** [review]
LGTM - merged.

**Pull request merged**